### DISCUSSION TITLE
Vertically align chat timestamp to sender

### DISCUSSION THREAD

**Discussion by cl8n** [ppy/osu#29199] [Ideas] [answered]
![image](https://github.com/user-attachments/assets/bcfbe04f-d7d9-4da2-bfde-dbb9ed8d23a4)

for me it becomes hard to quickly parse which message a timestamp is attached to like this, especially when there are multiple wrapping messages close together. not a design expert but I think the underlying issue has something to do with there being multiple kinds of alignment going on (timestamp is middle, but sender is top)

and probably just "this feels normal from other chat apps"

**Comment by peppy** [comment, marked as answer]
Will be resolved via https://github.com/ppy/osu/pull/29201.
